cmd/infra/powervs/create.go

```diff
@@ -1554,10 +1560,14 @@ func (infra *Infra) createTransitGateway(ctx context.Context, options *CreateInf
 	}
 
 	log(options.InfraID).Info("Creating Transit Gateway ...")
+	vpcRegion, err := regionutils.VPCRegionForPowerVSRegion(options.Region)
+	if err != nil {
+		return nil, fmt.Errorf("vpc region not found: %w", err)
```

Comment:
Pls update the error with the more context on where its occurring  and why

Reply:
```vpc region not found for supplied powervs region:%s error:%w. Please use correct values for PowerVs and VPC regions.```

Reply:
`respective vpc region not found for supplied powervs region:%s error:%w. Please use valid PowerVS and VPC regions`